integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -277,10 +275,13 @@ def save(self, commit: bool = True) -> Region:
             not self.instance.id and self.cleaned_data["duplicated_region"]
         )
 
+        hix_enabled = self.cleaned_data["hix_enabled"]
+
         # Save CustomModelForm
         region = super().save(commit=commit)
 
         if duplicate_region:
+            region.hix_enabled = False
```

Comment:
It would be nice to note why we decide to hard code it to disabled at this stage
```suggestion
            # Disable HIX during the duplication process,
            # to skip recalculation for already known content
            region.hix_enabled = False
```